src/main/java/core/basesyntax/service/StorageToStringService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface StorageToStringService {
```

Comment:
bad name for a class 
better to rename it to `ReportMakerService` or sth like this